### DISCUSSION TITLE
DYNAMITE Core Team Meeting May 3rd

### DISCUSSION THREAD

**Discussion by boeckera** [dynamics-of-stellar-systems/dynamite#385] [Announcements]
Hola @dynamics-of-stellar-systems/dynamite-maintainers,

we will meet again this Friday, same time (1pm), same place (Glenn's office or zoom). 
Unless anybody else wants to bring forth things to discuss, we will be going through the issues/pull requests.
Please react with an emoji to signal your participation.

Cheers,
Alina

**Comment by maindlt** [comment]
Maybe interesting to some of you: I noticed the VSC team is offering a free [Python for HPC](https://events.vsc.ac.at/e/Python4HPC-2024-05) online course (13.+15.+17.05.2024) and signed up for it, mainly to get an overview of "how to parallelize code for single and multi-node deployment as well as GPU offloading" ;-)
